app/templates/admin/bonnerManagement.html

```diff
@@ -32,12 +32,13 @@ <h3 class="accordion-header" id="headingOne">
             <!---------------- Cohort management ---------------->
             <div class="d-flex justify-content-center">
                 <div class="nav flex-column nav-pills me-3" id="v-pills-tab" role="tablist" aria-orientation="vertical">
-                {% set subTab = cohorts.keys()|last if subTab == "newest" else subTab %}
-                {% for year in cohorts.keys()|reverse %}
-                    {% set show = "active" if subTab|int == year else "" %}
-                    {% set aria = "true" if subTab|int == year else "false" %}
-                    <button class="nav-link {{show}}" id="v-pills-{{year}}-tab" data-bs-toggle="pill" data-bs-target="#v-pills-{{year}}" type="button" role="tab" aria-controls="v-pills-{{year}}" aria-selected="{{aria}}">{{year}} - {{year + 1}}</button>
-                {% endfor %}
+                    <button class="btn-success py-2 mb-2 rounded boarder-0" id="addCohort">Add New Cohort</button>
```

Comment:
I think we should probably use the default button styling for this button. The different styling sticks out a little bit here.

Reply:
Thanks for teaching me how to use bootstrap like a pro